apps/frontend/src/app/Data/Characters/KujouSara/index.tsx

```diff
@@ -250,7 +250,9 @@ const sheet: ICharacterSheet = {
           TenguJuuraiAmbush: {
             fields: [
               {
-                node: skillTenguAmbush_,
+                node: infoMut(skillTenguAmbush_disp, {
+                  name: st(`increase.atk`),
```

Comment:
shouldnt this just use keymap.info('atk')?

Reply:
ig so, then. did it to Bennett too.